### DISCUSSION TITLE
Move asdf-kotlin here

### DISCUSSION THREAD

**Discussion by missingcharacter** [asdf-community/.github#10] [General]
I'd like to move https://github.com/missingcharacter/asdf-kotlin here and then I'll update https://github.com/asdf-vm/asdf-plugins/blob/master/plugins/kotlin

**Comment by smorimoto** [comment]
I invited you, go ahead!

**Comment by missingcharacter** [comment]
It is here now :) https://github.com/asdf-community/asdf-kotlin, thanks @smorimoto I'll update https://github.com/asdf-vm/asdf-plugins/blob/master/plugins/kotlin next

**Comment by missingcharacter** [comment]
PR is here, but I have to figure out why `test_plugin.sh` think I'm modifying more than 1 plugin

**Comment by smorimoto** [comment]
It would be better to switch to GitHub Actions at the same time and replace the badge. (It eliminates the need to open a PR in the plugins repository multiple times.)

**Comment by smorimoto** [comment]
It certainly has failed, but you don't have to worry about it for the time being :)

**Comment by missingcharacter** [comment]
anyways, I updated `test_plugin.sh` and made my commit part of my PR to update kotlin location https://github.com/asdf-vm/asdf-plugins/pull/304, but if you want I can make it a different PR

**Comment by smorimoto** [comment]
Great! Yes, it should definitely be a different PR.

**Comment by missingcharacter** [comment]
New PR https://github.com/asdf-vm/asdf-plugins/pull/305

**Comment by missingcharacter** [comment]
I think we are good now! Thanks @smorimoto

**Comment by smorimoto** [comment]
Thank you, too!
